eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -231,22 +239,37 @@ jobs:
         customBuildArgs="$customBuildArgs --use-mono-runtime"
       fi
 
-      docker run --rm -v "$(sourcesPath):/vmr" -w /vmr $customRunArgs ${{ parameters.container }} ./build.sh --clean-while-building $(additionalBuildArgs) $customBuildArgs
+      if [[ ! -z '${{ parameters.crossRootFs }}' || ! -z '${{ parameters.overrideRid }}' ]]; then
+        extraBuildProperties="--"
+      fi
+
+      if [[ ! -z '${{ parameters.crossRootFs }}' ]]; then
+        customRunArgs="$customRunArgs -e CROSSCOMPILE=1 -e ROOTFS_DIR=${{ parameters.crossRootFs}}"
```

Comment:
I ran into issues setting CROSSCOMPILE=1, does this work for you? I wasn't setting PortableBuild=true, maybe that's why I hit issues?

Reply:
Oh, I hit that issue passing it to the `dotnet/dotnet` build script, never mind.